mirrord-agent/src/main.rs

```diff
@@ -160,6 +231,33 @@ async fn handle_peer_messages(
                 .send((peer_message.peer_id, file_request))
                 .await?;
         }
+        ClientMessage::OverrideEnvVarsRequest(OverrideEnvVarsRequest { filter_env_vars }) => {
```

Comment:
I think the name `OverrideEnvVarsRequest` is too "use case specific" while what really happens here is layer asking the agent for env vars, so I'd suggest to rename it to `GetEnvVars`

Reply:
`GetEnvVars` makes sense, I've tried out so many bad names for it.